romanesco/format/__init__.py

```diff
@@ -248,6 +248,6 @@ def import_default_converters():
     import_converters([os.path.join(cur_path, t) for t in [
         "r", "table", "tree",
         "string", "number", "image",
-        "boolean", "geometry"]])
+        "boolean", "geometry", "netcdf"]])
```

Comment:
Nice. In the future, I am assuming that we may need to have some mechanism for plugins? so that this list won't grow too big?